package.json

```diff
@@ -108,7 +109,7 @@
     "**/@babel/helpers": "^7.27.0",
     "**/@babel/runtime": "^7.27.0",
     "**/@babel/runtime-corejs3": "^7.27.0",
-    "**/@babel/traverse": "^7.23.2",
+    "**/@babel/traverse": "^7.27.0",
```

Comment:
This was upgraded due to some runtime errors I saw due to the incompatible babel versions. This is safe to upgrade to 7.27.0 according to: https://github.com/advisories/GHSA-67hx-6x53-jw92